app/views/household_introduction/edit.html.erb

```diff
@@ -0,0 +1,73 @@
+<% content_for :header_title, 'Introduction' %>
+
+<div class='form-card'>
+  <header class='form-card__header'>
+    <div class='step-section-header'>
+      <div class="step-section-header__icon illustration illustration--section-1"></div>
+      <div class='step-section-header__subhead'>
+        There are 4 sections you need to complete to submit a full application.
+      </div>
+    </div>
+  </header>
+
+  <div class='form-card__content'>
+    <div class='slab slab--not-padded'>
+      <div class='card card--narrow'>
```

Comment:
I think if you remove all of the container divs in the PR with the class `card card--narrow` these will look a little better and ready-to-go.  I *think*! Give it a go?